releasenotes/notes/32949.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+- 32949
+releaseNotes:
```

Comment:
can we add an upgradeNote as well, explaining an exact YAML example of before and after IOP?